setup.py

```diff
@@ -12,18 +12,17 @@
     license='MIT',
     packages=find_packages(exclude=('tests',)),
     keywords='neo4j django plugin neomodel',
-    install_requires=['neomodel>=3.2.5', 'django>=1.9'],
+    install_requires=['neomodel>=4.0.1', 'neo4j-driver==4.1.1', 'pytz>=2020.1', 'django==2.2'],
```

Comment:
Can't we set `django>=2.2` or does a later version brake it? neomodel pulls in neo4j-driver right so no need to include that here

Reply:
thanks